asmcomp/emitaux.ml

```diff
@@ -198,9 +198,9 @@ let emit_frames a =
       a.efa_8 (List.length dbg);
       List.iter (fun Debuginfo.{alloc_words;_} ->
         (* Possible allocations range between 2 and 257 *)
-        assert (2 <= alloc_words &&
+        assert ((alloc_words = 0) || (2 <= alloc_words &&
                 alloc_words - 1 <= Config.max_young_wosize &&
-                Config.max_young_wosize <= 256);
+                Config.max_young_wosize <= 256));
         a.efa_8 (alloc_words - 2)) dbg;
```

Comment:
The assertion that alloc_words is between 2 and 257 is to make sure that `efa_8 (alloc_words - 2)` doesn't overflow. This changes the assertion without changing the format, so this isn't going to work.

Why do polling points need a Dbg_alloc frame anyway?